src/renderer/dx/DxFontRenderData.cpp

```diff
@@ -809,15 +803,22 @@ void DxFontRenderData::_BuildFontRenderData(const FontInfoDesired& desired, Font
 
     const auto fullPixelAscent = ceil(ascent + halfGap);
     const auto fullPixelDescent = ceil(descent + halfGap);
-    lineSpacing.height = fullPixelAscent + fullPixelDescent;
-    lineSpacing.baseline = fullPixelAscent;
+    const auto defaultHeight = fullPixelAscent + fullPixelDescent;
+    const auto lineHeight = desired.GetCellHeight().Resolve(defaultHeight, dpiF, heightDesired, widthAdvanceInPx);
+    const auto baseline = fullPixelAscent + (lineHeight - defaultHeight) / 2.0f;
+
+    lineSpacing.height = roundf(lineHeight);

```

Comment:
This isn't going to result in a different outcome for users who haven't customized cellHeight, right?

Reply:
Yeah, because `defaultHeight` is an integer and that's what `Resolve()` returns if no cellHeight is set.